lib/ansible/modules/cloud/google/gcp_compute_instance.py

```diff
@@ -434,13 +434,13 @@
         type: bool
   status:
     description:
-    - 'The status of the instance. One of the following values: PROVISIONING, STAGING,
-      RUNNING, STOPPING, SUSPENDING, SUSPENDED, and TERMINATED.'
-    - As a user, use RUNNING to keep a machine "on" and TERMINATED to turn a machine
-      off .
-    - 'Some valid choices include: "PROVISIONING", "STAGING", "RUNNING", "STOPPING",
-      "SUSPENDING", "SUSPENDED", "TERMINATED"'
+    - 'The desired state of the instance. One of the following values:
+      RUNNING, STOPPED, SUSPENDED, and TERMINATED.'
+    - As a user, use RUNNING to keep a machine "on" and TERMINATED or STOPPED
+      to turn a machine off .
+    - 'Some valid choices include: "RUNNING", "STOPPED", "SUSPENDED", "TERMINATED"'
```

Comment:
```suggestion
```

Reply:
These states are all repeated under choices so don't need to add them here except for where you are explaining what each means.

Reply:
Agree. Fixed